CONTRIBUTING.md

```diff
@@ -15,6 +15,22 @@ By making a contribution to this project, I certify that:
   person who certified (a), (b) or (c) and I have not modified it.
 
 
+## Contribution Guidelines
+
+While you're working on contributions, and submitting pull requests (PRs) to add those contributions, to this project, please keep these guidelines in mind. We _expect_ submitted code, and its associated PR, to follow these standards. Please follow them so you don't have to lose your time fixing your PRs, and so we don't have to lose time explaining why.
+
+#### Code
+* __One function per PR:__ Please add only one function or feature with one PR. The project is easier to maintain when we can easily grasp everything a PR is doing, and can reference each feature by individual PR in the future.
+
+#### Pull Requests
+* __PR Timeout__: If a PR has merge conflicts or one or more downvotes and the author of a PR doesn't respond to inquiries or requests posted to the PR issue after 72 hours, the PR will be closed.
+* __Controversial PRs:__ If a PR is controversial, it will be discussed by Contributors in the issue, and, if required, in the bi-weekly Website Working Group (WG) meeting.
+* __Two contributor votes for merge:__ Two WG members of Contributor status will need to approve, or +1, your contribution in order for it to be accepted and merged into the main repo.
```

Comment:
To keep it more general (and get it merged), why not drop the "two" and just go with this instead:
> * __Contributor votes for merge:__ WG members of Contributor status will need to approve, or +1, your contribution in order for it to be accepted and merged into the main repo.

It still explains the process without being too fixated on 1/2/n approvals. We can still make this more explicit later on (if needed).